packages/cozy-pouch-link/src/helpers.js

```diff
@@ -1,13 +1,34 @@
 import startsWith from 'lodash/startsWith'
 
-export const withoutDesignDocuments = res => {
-  const rows = res.rows.filter(doc => !startsWith(doc.id, '_design/'))
+// https://github.com/pouchdb/pouchdb/issues/7011
+const LIMIT_BUG = 999
+const ADAPTER_WITH_LIMIT_BUG = ['cordova-sqlite', 'websql']
 
-  return {
-    ...res,
-    rows,
-    total_rows: rows.length
+export const find = async (db, options = {}) => {
+  if (options.selector === undefined) {
+    options.selector = { _id: { $exists: true } }
   }
+  const limit = options.limit
+
+  if (ADAPTER_WITH_LIMIT_BUG.includes(db.adapter) || options.sqlite) {
```

Comment:
Why is there options.sqlite ? What is the difference with db.adapter == cordova-sqlite  ?

Reply:
it's for simulate sqlite on test...